Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -19,6 +22,11 @@ def linkage
     ARGV.kegs.each do |keg|
       ohai "Checking #{keg.name} linkage" if ARGV.kegs.size > 1
       result = LinkageChecker.new(keg)
+
+      # Force a flush of the 'cache' and check dylibs if the `--rebuild`
+      # flag is passed
+      result.check_dylibs if ARGV.include?("--rebuild")
```

Comment:
Could rename the method `flush_cache_and_check_dylibs` and kill comment entirely